README.md

```diff
@@ -136,6 +136,12 @@ An array with fixtures (defaults). Each fixture has to be an object with a `labe
 
 You can also add a `className` key to a fixture. This class will be applied to the fixture item.
 
+### maxFixtures
+Type: `Number`
+Default: `10`
+
+Maximum number of fixtures to render.
```

Comment:
In the demo I provided below, I have defined 13 fixtures, but only 10 are rendered. This configuration solves that problem.